### ISSUE TITLE
batch prediction multiple datasets prediction export

### ISSUE THREAD

**Issue by czhci** [ilastik/ilastik#827]
batch prediction - when multiple datasets are loaded to be processed, "choose settings" changes settings for every dataset to be batch processed. e.g. when the file name is manually specified and the {nickname} or {dataset_dir} is gone, then all (or some) of the dataset will have the same export name, and this causes error when trying to write to the same location. Maybe some warning message could be displayed to ask for changes when this happens? Or a better specification in the choose settings to prevent this?

**Comment by akreshuk** [comment]
@stuarteberg , i think this was fixed in the batch processing redesign?

**Comment by stuarteberg** [comment]
No, I don't think this was changed.  If the user omits the `{dataset_dir}` magic keyword, then all results will be (over)written into a single file repeatedly.
